disconnect-entitylist.json

```diff
@@ -920,16 +920,6 @@
             "dmtry.com"
         ]
     },
-    "AdOn Network": {
-        "properties": [
-            "adonnetwork.com",
-            "dashboardad.net"
```

Comment:
Hmm ... I don't see `dashboardad.net` in the entity list at all anymore? Did this get lost?